draft-wang-ppm-differential-privacy.md

```diff
@@ -451,9 +465,81 @@ class DpMechanism:
 
 > TODO: Specify a Gaussian sampler
 
-## Randomized Response {#rr}
+## Binary Randomized Response {#rr}
+
+The binary randomized response mechanism specified here aims to provide the
+DP guarantee at the Client level. It is the same as the one specified in
```

Comment:
Be specific ("approximate-DP" is more meaningful than "DP guarantee" because we defined it previously) and specify the trust model.
```suggestion
good approximate-DP in the OAMC trust model. It is the same as the one specified in
```